package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "currencyinfo",
-  "version": "0.0.1",
-  "description": "",
-  "author": "",
+  "version": "4.0.0",
+  "description": "Self-hosted crypto and fiat currency rates service provider",
+  "author": "ADAMANT Foundation <devs@adamant.im> (https://adamant.im)",
   "private": true,
-  "license": "UNLICENSED",
+  "license": "GPL-3.0",
```

Comment:
Is the format correct? E.g. adamant-im repository uses `GPLv3`.
https://github.com/Adamant-im/adamant-im/blob/master/package.json#L5